index.html

```diff
@@ -1,151 +1,211 @@
-                <li><a href="#" class="fa fa-linkedin"></a></li>
-                <li><a href="#" class="fa fa-google"></a></li>
-                <li><a href="#" class="fa fa-github"></a></li>
-                <li><a href="#" class="fa fa-javascript"></a></li>
-            </ul>
-        <button class="resume">Get My Resume</button>
-        <h3 class="languages">Languages</h3>
-        
-            <img src="images/skills/html.svg" alt="html">
-            <img src="images/skills/css.svg" alt="css">
-            <img src="images/skills/js.svg" alt="js">
-        
-        <h3 class="frameworks">Frameworks&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
-            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;></h3>
-        <hr>
-        <h3 class="frameworks">Skills&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
-            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;></h3>
-        <hr>
+      <h1 class="about-myself-title">
+        About <br />
+        Myself
+      </h1>
+      <p class="about-myself-content">
+        Hello I am a software develper. I can help you build a product, feature
+        or website. Look through some of my work and expereince. If you like
+        what you see and have a project you need coder, don't husitate to
+        contact me.
+      </p>
+      <h3 class="links">LET'S CONNECT</h3>
+      <ul class="socials">
+        <li><a href="#" class="fa fa-facebook"></a></li>
+        <li><a href="#" class="fa fa-twitter"></a></li>
+        <li><a href="#" class="fa fa-linkedin"></a></li>
+        <li><a href="#" class="fa fa-google"></a></li>
+        <li><a href="#" class="fa fa-github"></a></li>
+        <li><a href="#" class="fa fa-javascript"></a></li>
+      </ul>
+      <button class="resume">Get My Resume</button>
+      <h3 class="languages">Languages</h3>
+
+      <img src="images/skills/html.svg" alt="html" />
+      <img src="images/skills/css.svg" alt="css" />
+      <img src="images/skills/js.svg" alt="js" />
+
+      <div class="frameworks">
+        <div class="framework1">Framework</div>
+        <div class="greater-tag">></div>
+      </div>
+      <hr>
+      <div class="frameworks">
+        <div class="framework1">Skills</div>
+        <div class="greater-tag">></div>
+      </div>
+      <hr>
     </section>
-</body>
-</html>
\ No newline at end of file
+    <footer class="contact">
+      <h1 class="contact-title">Contact me</h1>
+      <p class="contact-content">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/{form_id}" method="POST" class="my-form">

```

Comment:
- [ ] Hello @ezraermy, good job so far 👏. But the form submission is still not working.
 
![Screenshot from 2022-11-01 23-09-05](https://user-images.githubusercontent.com/89228743/199359252-cd54cdf1-cfa5-428e-905f-dcffe9a2ea9d.png)

- [x] What you need first is to create your account at [formspree](https://formspree.io/html/) and then generate the `form_id` by following the steps. 
- [x] In your `action` value replace `{form_id}` with the `id` you generated from  [formspree](https://formspree.io/html/). 👌
when you got the `id`, your `action` should be simmilar to something like this `action="https://formspree.io/f/hgkd1254us"`
- [x] Lastly, please make that a user can successfully submit a `form`. After that is done you can proudly submit it for review. 🤗

Reply:
## How to get the `action` value ?
[Demo video](https://user-images.githubusercontent.com/89228743/199359877-b51846d3-16b3-488d-92c2-a992ecee2b21.webm)